third_party/gtest/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ include(FetchContent)
 FetchContent_Declare(
     googletest
     GIT_REPOSITORY https://github.com/google/googletest.git
-    GIT_TAG        release-1.11.0
+    GIT_TAG        main
```

Comment:
I think it's better to tag a fixed version. Do you need a more recent version?

Reply:
My bad, that should not be part of the PR